Application/Features/BookingFeature/Commands/UpdateBookingCommand.cs

```diff
@@ -66,8 +66,8 @@ public async Task<UpdateBookingCommandResponse> Handle(UpdateBookingCommandReque
                 throw new NotFoundException(nameof(booking), request.Id);
             }
 
-            EnsureWorkplaceIsFree(request.WorkplaceId, request.StartDate, request.EndDate);
-            EnsureUserHasNotBookingThisTime(request.TelegramId, request.StartDate, request.EndDate);
+            await EnsureWorkplaceIsFree(request.WorkplaceId, request.StartDate, request.EndDate);
+            await EnsureUserHasNotBookingThisTime(request.TelegramId, request.StartDate, request.EndDate);
```

Comment:
Name this method as EnsureUserHasNotBookingThisTimeAsync

Reply:
done